src/lib/Contracts.ts

```diff
@@ -256,6 +259,12 @@ export class Contracts {
       { contract: this.doubleExponentInterestSetter, json: doubleExponentInterestSetterJson },
       { contract: this.wethPriceOracle, json: wethPriceOracleJson },
       { contract: this.daiPriceOracle, json: daiPriceOracleJson },
+      { contract: this.saiPriceOracle, json: daiPriceOracleJson, overrides: {
+        1: '0x787F552BDC17332c98aA360748884513e3cB401a',
+        42: '0x8a6629fEba4196E0A61B8E8C94D4905e525bc055',
```

Comment:
Is this right? It says this is the address of `AdminImpl` on mainnet?

Reply:
Since the same address is used to deploy on kovan and mainnet, and since the contract address is deterministic based off the nonce, sometimes the addresses overlap unfortunately